src/pip/_internal/wheel.py

```diff
@@ -741,6 +741,7 @@ def build(self, session, autobuilding=False):
 
         buildset = []
         for req in reqset:
+            ephem_cache = False
```

Comment:
bike-shedding: I'll suggest painting this as `use_ephem_cache`; I won't push if anyone disagrees.